src/opts.rs

```diff
@@ -78,11 +81,34 @@ pub struct Opts {
     /// Disable generating template solve script
     #[structopt(long)]
     pub no_template: bool,
+
+    /// Kernel-land initialization
+    #[structopt(long)]
+    pub ker: bool,
+
+    /// Name of vmlinux binary
+    #[structopt(long)]
+    #[setters(generate)]
+    pub vmlinux: Option<PathBuf>,
+
+    /// Name of bzImage file
+    #[structopt(long)]
+    #[setters(generate)]
+    pub bzimage: Option<PathBuf>,
+
+    /// Disable extraction of vmlinux from bzImage
+    #[structopt(long)]
+    pub no_extract_vmlinux: bool,
+
+    /// Disable patching vmlinux with vmlinux-to-elf
+    #[structopt(long)]
+    pub no_patch_vmlinux: bool,
 }
 
 impl Opts {
     /// Print the locations of known files (binary, libc, linker)
     pub fn print(&self) {
+        println!("Found following {}-land challenge binaries:", if self.ker { "kernel" } else {"user"});
```

Comment:
Pretty long line. And double spaces on line 124. Consider running rustfmt on changed files